channels/cliprdr/client/cliprdr_main.c

```diff
@@ -740,8 +740,18 @@ static UINT cliprdr_client_format_list(CliprdrClientContext* context,
 	    freerdp_settings_get_uint32(context->rdpcontext->settings, FreeRDP_ClipboardFeatureMask);
 	CLIPRDR_FORMAT_LIST filterList = cliprdr_filter_local_format_list(formatList, mask);
 	if (filterList.numFormats == 0)
-		return CHANNEL_RC_OK;
+	{
+		WLog_DBG(TAG, "forbidden direction local->remote, also need send empty file list");
```

Comment:
It is right, that it needs to be possible to send empty format lists (e.g. on the initialization sequence of cliprdr, where the client needs to send a format list as part of that sequence, but also when the selection did indeed change and no supported formats are on the selection list).

However, this message is not necessarily true. There are situations, where it matches the situation, but not always (see the aforementioned situations).
So, just remote this message